pkg/cloud_provider/file/file.go

```diff
@@ -1039,11 +1051,14 @@ func shareURI(project, location, instanceName, shareName string) string {
 	return fmt.Sprintf(shareURIFmt, project, location, instanceName, shareName)
 }
 
+// We support endpoints w/ http:// and without, so add in http:// if it's
```

Comment:
Same thing here. Ideally we would just only support full endpoint names to get rid of this ugliness. I'd need to see where this is getting passed w/o the prefix and update that, if someone can point me to that :)